common/src/main/java/com/wynntils/commands/WynntilsCommand.java

```diff
@@ -4,36 +4,319 @@
+
+                            if (foundFeature.isEmpty()) return Collections.emptyIterator();
+
+                            return Arrays.stream(foundFeature.get().getConfigFields())
+                                    .map(Field::getName)
+                                    .iterator();
+                        },
+                        builder);
+
+        // Feature specified, config option is not, print all configs
+        // If a feature and config field is specified, print field specific info
+        // /wynntils config get <feature>
+        // /wynntils config get <feature> <field>
+        getConfigArgBuilder.then(Commands.argument("feature", StringArgumentType.word())
+                .suggests(featureSuggestionProvider)
+                .then(Commands.argument("config", StringArgumentType.word())
+                        .suggests(featureConfigSuggestionProvider)
+                        .executes(this::getSpecificConfigOption))
+                .executes(this::listAllConfigOptions));
+
+        // /wynntils config set <feature> <field> <newValue>
+        setConfigArgBuilder.then(Commands.argument("feature", StringArgumentType.word())
+                .suggests(featureSuggestionProvider)
+                .then(Commands.argument("config", StringArgumentType.word())
+                        .suggests(featureConfigSuggestionProvider)
+                        .then(Commands.argument("newValue", StringArgumentType.greedyString())
+                                .executes(this::changeFeatureConfig))));
+
         dispatcher.register(Commands.literal("wynntils")
                 .then(Commands.literal("help").executes(this::help))
                 .then(Commands.literal("discord").executes(this::discordLink))
                 .then(Commands.literal("donate").executes(this::donateLink))
                 .then(Commands.literal("reload").executes(this::reload))
                 .then(Commands.literal("version").executes(this::version))
+                .then(Commands.literal("config").then(getConfigArgBuilder).then(setConfigArgBuilder))
+                .then(Commands.literal("feature").then(Commands.literal("list").executes(this::listFeatures)))
                 .executes(this::help));
     }
 
+    private int listFeatures(CommandContext<CommandSourceStack> context) {
+        Set<Feature> features = FeatureRegistry.getFeatures().stream().collect(Collectors.toUnmodifiableSet());
+
+        MutableComponent response = new TextComponent("Currently registered features:").withStyle(ChatFormatting.AQUA);
+
+        for (Feature feature : features) {
+            String longFeatureName = Arrays.stream(CaseFormat.LOWER_CAMEL
+                            .to(CaseFormat.LOWER_UNDERSCORE, feature.getShortName())
+                            .split("_"))
+                    .map(StringUtils::capitalize)
+                    .collect(Collectors.joining(" "));
+            response.append(new TextComponent("\n-").withStyle(ChatFormatting.GRAY))
```

Comment:
```suggestion
            response.append(new TextComponent("\n - ").withStyle(ChatFormatting.GRAY))
```

Make the list a bit less compact.